### DISCUSSION TITLE
Setting env TYPE: ValheimPlus | ERROR

### DISCUSSION THREAD

**Discussion by TheoMarque** [mbround18/valheim-docker#461] [Help!]
As mentioned in topic, not working. Unix server is downloaded but... not installed. I manually install it in image using docker exec -it valheim bash using valheimplus github repository for unix server. After that i must change startup script name into default. After this - all worked fine. But Is a problem wit unpacking and install procedure for valheimplus.

**Comment by mbround18** [comment]
@TheoMarque  what version of the image are you running? This was a problem that was fixed in 1.4.8+

**Reply by TheoMarque** [reply]
mbround18/valheim                             latest              deef4c1771f1        2 weeks ago         798MB

**Reply by mbround18** [reply]
@TheoMarque please update to tag 1.4.10

**Reply by TheoMarque** [reply]
Updated, seems problem was solved. Why latest image not "latest"?

**Reply by mbround18** [reply]
> Updated, seems problem was solved. Why latest image not "latest"?

It's just a misunderstanding, latest means it will download the latest image at that point in time. It won't pull a new image unless a few configurations on your compose are set or you run a pull for the latest version.
